pinot-common/src/main/java/org/apache/pinot/common/utils/config/QueryOptionsUtils.java

```diff
@@ -190,4 +191,25 @@ public static Integer getGroupTrimThreshold(Map<String, String> queryOptions) {
     String groupByTrimThreshold = queryOptions.get(QueryOptionKey.GROUP_TRIM_THRESHOLD);
     return groupByTrimThreshold != null ? Integer.parseInt(groupByTrimThreshold) : null;
   }
+
+  // Sample input: k1:1/k1:2/k1:3/k2:4/k2:5/k2:6
+  // Sample output: {k1=[1, 2, 3], k2=[4, 5, 6]}
+  public static Map<String, Set<Integer>> getColumnPartitionMap(Map<String, String> queryOptions) {
```

Comment:
Couldn't we use other standard format (like json) here? Or json without quotes or something like that. Adding more formats makes it complex for the writer and introduce new bugs. For example, this method does not allow to use `:` or `/` in the values.

By using json without quotes we can implement the method with jackson (changing the config as explained [here](https://stackoverflow.com/questions/28788331/creating-json-without-quotes)) and the input would be something like:

```js
{k1: [1, 2, 3], k2: [4,5,6]}
```

Reply:
+1, just curious why this format?